ui/selectors/multi-srp/multi-srp.ts

```diff
@@ -109,6 +109,7 @@ export const getSnapAccountsByKeyringId = createDeepEqualSelector(
   (accounts, keyringId) => {
     return accounts.filter(
       (account: InternalAccount) =>
+        account.metadata.snap?.id &&
```

Comment:
Same here probably? Also, couldn't we use this selector in the `usHdKeyringsWithSnapAccounts` hook? That would avoid this kind of duplicated logic IMO.